tc/core/tc2halide.cc

```diff
@@ -492,20 +496,25 @@ Expr reductionUpdate(Expr e) {
   return Call::make(e.type(), kReductionUpdate, {e}, Call::Intrinsic);
 }
 
+// translate a single TC comprehension/statement to Halide component.
+// funcs, bounds, reductions will be populated
 void translateComprehension(
-    const lang::Comprehension& c,
+    const lang::Comprehension& comprehension,
```

Comment:
while reading the code, it was confusing to me to read `c` so I made it more descriptive to facilitate code readability. Hope this is okay, otherwise I can revert too.

Reply:
+1, one character names are for iteration variables of short loops